content/en/docs/zero-code/obi/quickstart/cpp.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Quickstart: instrument a C/C++ service with OBI'
+menuTitle: C/C++ quickstart
+description:
+  Learn how to quickly set up and run OBI to instrument a C/C++ service
+weight: 2
+---
+
+# Quickstart: instrument a C/C++ service with Beyla
+
+## 1. Run an instrumentable C/C++ service
+
+Run an instrumentable C/C++ service or download and run a simple example
+[C++ HTTP service](https://github.com/grafana/beyla/tree/main/examples/quickstart/cpp).
+
+```
+curl -OL https://raw.githubusercontent.com/grafana/beyla/main/examples/quickstart/cpp/httplib.h
+curl -OL https://raw.githubusercontent.com/grafana/beyla/main/examples/quickstart/cpp/quickstart.cpp
+g++ -std=c++11 quickstart.cpp -o quickstart && ./quickstart
+```
```

Comment:
@open-telemetry/ebpf-instrumentation-approvers it looks like those examples files have not (yet) been migrated to the repo?

Reply:
removed